docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -15,8 +15,8 @@ swap. The table metadata file tracks the table schema, partitioning
 configuration, custom properties, and snapshots of the table contents.
 
 Iceberg data files are stored in either Parquet, ORC, or Avro format, as
-determined by the `format` property in the table definition.  The default
-`format` value is `PARQUET`.
+determined by the `iceberg.file-format` property in the catalog properties file.
```

Comment:
Maybe we need to also say that the table property is format and the column for the metadata is file_format .. its a mess..

Reply:
I think we should rather stick to the old version of this sentence.
We can have `icebergb.file-format`   set to PARQUET, but create explicitly a table with `format` AVRO.

Reply:
The old is not really correct either, but I am moving these details in later sections.